umm-spec-lib/src/cmr/umm_spec/migration/version/granule.clj

```diff
@@ -127,6 +127,14 @@
     (assoc related-url :Type "GET DATA")
     related-url))
 
+(defn- specify-metadata
```

Comment:
nit: I would call this function something like `set-version` or `update-version`.

Reply:
I did not go back and change all the older tests to use my new fancy function. I will if you would like me to.